src/components/Training/Approach.js

```diff
@@ -13,29 +15,11 @@ const Approach = ({ title, content, formats }) => (
       bottom={{ smallPhone: 3, desktop: 4 }}
```

Comment:
This whole padding bit should be deleted, I think.

Designs for this page (only the 72px coming from the TitleAndList component itself):
![image](https://user-images.githubusercontent.com/15656538/51908457-d1618400-23c1-11e9-840a-2d1a80a5a6f4.png)

Your preview branch (has extra padding on top of the 72px):
![image](https://user-images.githubusercontent.com/15656538/51908270-441e2f80-23c1-11e9-9655-c15240ddff34.png)

Also here:
![image](https://user-images.githubusercontent.com/15656538/51908941-1c2fcb80-23c3-11e9-947b-582029619e4c.png)

Reply:
You're totally right, missed it!